### DISCUSSION TITLE
Random crashes (maybe while playing with video background?)

### DISCUSSION THREAD

**Discussion by puyoxyz** [ppy/osu#15993] [Q&A] [answered]
osu!lazer is crashing randomly, with nothing interesting in the logs (the ones in `~/.local/share/osu/logs/`), and just a "zsh: segmentation fault" when it crashes if I run it in the terminal ([full log, but i don't think the vdpau/nv-glx errors have anything to do with the crashes since the video does play, and they appear way before the crash/every time a video starts](https://paste.puyo.xyz/nuliponafe))

~~it seems that it might be happening when i play with video backgrounds enabled (and a map with a video)? not sure about this, i'll do a bit more testing (but it does seem to be the case)~~ see edit

i'm on wayland, but running osu!lazer with `SDL_VIDEODRIVER=x11` (so it runs on xwayland) because running it without it (or with `SDL_VIDEODRIVER=wayland`) [makes it not appear but still have audio and still appear in my dock](https://github.com/ppy/osu/discussions/15625)

artix linux, gnome 41.2, intel i7-8700 + nvidia gtx 1060 6gb

edit: nevermind!! it has nothing to do with video backgrounds, seems i was just getting lucky when not having them on and not getting lucky when having them on ?

**Comment by frenzibyte** [comment, marked as answer]
Please attach the game logs (see https://github.com/ppy/osu/discussions/15188).

**Reply by puyoxyz** [reply]
oops sorry i forgot

runtime.log -> https://paste.puyo.xyz/iqepamasog

performance.log -> https://paste.puyo.xyz/upudetesun

network.log -> https://paste.puyo.xyz/cotipepexe

database.log -> https://paste.puyo.xyz/qezuyobeve

**Reply by peppy** [reply]
No crashes showing in logs. Must be somewhere else on your system.

**Reply by puyoxyz** [reply]
> No crashes showing in logs. Must be somewhere else on your system.

well duh of course it not log it it's a seg fault it deaded

**Reply by peppy** [reply]
Right. This is out of our area of concern then. It's likely going to be something at a system or driver level.
